backend/app/api/endpoints/adapter/tasks.py

```
@@ -684,3 +684,213 @@ def delete_task_services(
+                    detail=f"Failed to get workspace files: {response.text[:200]}",
+                )
+
+            data = response.json()
+            return WorkspaceFilesResponse(
+                files=data.get("files", []),
+                total_files=data.get("total_files", 0),
+                total_size=data.get("total_size", 0),
+            )
+
+    except httpx.TimeoutException:
+        logger.error(f"Timeout getting workspace files for task {task_id}")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)